getting_started/kubenetes.md

```diff
@@ -23,118 +23,168 @@ contains instructions for launching various versions of Kubernetes
 For trying out Kontain with Kubernetes, you can launch Minikube with Docker Desktop [view instructions here](/appendix/minikube/).
 
 ```shell
-$ kind create cluster
+ kind create cluster
 ```
 
 or Minikube:
 
 ```shell
-$ minikube start --container-runtime=containerd --driver=docker --wait=all
+ minikube start --container-runtime=containerd --driver=docker --wait=all
 ```
 ===
 
-## Install Kontain on Kubernetes nodes using a Daemonset install
-Once you have ensured that KVM has been enabled on your worker node(s), deploy Kontain Runtime using the Kubernetes client CLI.
+
+## Enabling Kontain on Cloud Kubernetes/Openshift
++++ Azure AKS
+Deploy Kontain Runtime using the Kubernetes client CLI and Kontain Deamonset
 
 ```shell
-# we install Kontain using a Daemonset
-$ kubectl apply -f https://raw.githubusercontent.com/kontainapp/guide-examples/master/infra/kustomize_outputs/km.yaml
+ kubectl apply -f https://raw.githubusercontent.com/kontainapp/km/current/cloud/k8s/deploy/azure-aks/kontain-install.yaml
 ```
 
 The above yaml contains the Kontain runtime class and the install scripts for the Daemonset to be installed.
 
-**Important note:**
-Please note that AWS EKS and Google Cloud GKE do allow KVM to be enabled.  For installing Kontain on these versions of Kubernetes please refer to the below.
++++ AWS EKS
+To create EKS cluster with Kontain:
 
-[!ref text="To install Kontain on AWS, GKE nodes"](/getting_started/kubenetes/#aws-eks-with-containerd-and-google-cloud-gke)
+1. Create cluster on EKS using Kontain-enabled AMI (kontain-eks-node-1.22-*version*)
```

Comment:
We have too many AMIs and the names are confusing.  There are many versions of kontain-amazon-eks-node and amazon-eks-node.
We will need to specify exactly which version for him to use, and delete the rest, else he will not know which one to use in the console.  Or we will need to leave behind just 1.

Reply:
Made some tiny fixes as well, if we can finalize which version of AMI for him to use and possibly delete/de-register the rest, then we will be good to go.

Reply:
A new version will be built every time we do a release. Maybe we can do a script that gives a ‘current’ version from current-release.txt? Also, good point about region. We can add propagating Ami  to list of regions (to be decided, which) to the release.yaml . Let me know which way we should go , or both

Reply:
I like the script idea of giving the 'current' version, which can also probably be built into the EKS creation script so it by default uses the 'current' one in the region it is being launched.  No problem if we have to do this after the release is cut, because I think the script will keep evolving independent of release.

Also, think if we can copy (using aws copy-image maybe) it to maybe the us-east-1, us-east-2, us-west-1, us-west-2 we should be fully covered.

We do need to make a decision on whether we "unregister" all current AMIs so we start fresh.  Currently, its showing 15 of them in the marketplace.  We should probably have a policy of keeping last 3 or something like that.

Reply:
Yes there are a lot of various AMIs there. We do need to clean up the today's set of AMIs. We'll do that as soon as we generate the three AMIs for v0.9.9 release. 

Simply search for `kontain-eks-node` in AMIs. For now there will be only one. So in a few month maybe there will be two or three - very easy to eyeball and see the latest. Maybe explicitly the search mention that in the doc.

EKS cluster needs to be in the same region as the AMI. I guess they can copy it as needed, just mention that in the doc too.

Reply:
I’m updating document with those two extra CLI command - one to find AMI and one to copy to region you want if not there.

Reply:
It looks like if we make it public and have a link similar to below in docs, it should make it much easier for a user to identify which image to use in his UI to create EKS clusters with that AMI (again, this is only for UI):
https://us-east-2.console.aws.amazon.com/ec2/v2/home?region=us-east-2#ImageDetails:imageId=ami-04775489e637fe281

Reply:
"Simply search for kontain-eks-node in AMIs" and probably filter by date.

Let's mention this in the docs. That should be enough instead of going further.